maintainers/scripts/luarocks-packages.csv

```diff
@@ -120,6 +120,7 @@ nlua,,,,,,teto
 nui.nvim,,,,,,mrcjkb
 nvim-cmp,https://raw.githubusercontent.com/hrsh7th/nvim-cmp/main/nvim-cmp-scm-1.rockspec,,,,,
 nvim-nio,,,,,,mrcjkb
+orgmode,,,https://luarocks.org/dev,,,
```

Comment:
question: Is there a reason for using the dev version here? Afaict, orgmode has regular releases.
Granted, the last one is a few months ago, but perhaps we could suggest they use a release-please workflow for more frequent releases (they already use conventional commits).

Reply:
To keep parity with the existing vimPlugin and not cause a breaking change.

Reply:
https://github.com/nvim-orgmode/orgmode/issues/828

Reply:
longterm we want to target official releases I think but right now as it's already quite a bit of work what @khaneliman has started doing, I would go with whatever he prefers (dev or last release): it's easy enough to change afterwards.